client/src/components/JurisdictionAdmin/DownloadBatchTallySheetsButton/BatchTallySheet.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import BatchTallySheet from './BatchTallySheet'
+
+jest.mock('@react-pdf/renderer', () => ({
+  ...jest.requireActual('@react-pdf/renderer'),
+
+  // Mock @react-pdf/renderer to generate HTML instead of PDF content for easier testing
+  Document: jest.fn(({ children }) => <div>{children}</div>),
+  Page: jest.fn(({ children }) => <div>{children}</div>),
+  Text: jest.fn(({ children }) => <div>{children}</div>),
+  View: jest.fn(({ children }) => <div>{children}</div>),
+}))
+
+test('BatchTallySheet renders expected PDF content', async () => {
+  const { container } = render(
+    <BatchTallySheet
+      auditBoardName="Audit Board 1"
+      batchName="Batch 1"
+      choices={[
+        { id: 'choice-1', name: 'Choice 1', numVotes: 0 },
+        { id: 'choice-2', name: 'Choice 2', numVotes: 0 },
+        { id: 'choice-3', name: 'Choice 3', numVotes: 0 },
+      ]}
+      jurisdictionName="Jurisdiction 1"
+    />
+  )
+  expect(container).toMatchSnapshot()
```

Comment:
Agreed! We even have a way to snapshot the resulting PDF: https://github.com/votingworks/arlo/blob/1dfea479613546d5ee8210d06d297958f895b5aa/client/src/components/JurisdictionAdmin/generateSheets.test.tsx#L60. I think it's nice to have a visual snapshot diff for PDFs so that you can really make sure the styling works as expected

Reply:
Ooh very cool!

I just did a quick investigation into generating visual snapshots for PDFs generated by `@react-pdf/renderer` but was reminded why I did my [HTML mocking](https://github.com/votingworks/arlo/blob/950c9fced920c1d03e99cdd99e052c371f5aa01e/client/src/components/JurisdictionAdmin/DownloadBatchTallySheetsButton/BatchTallySheet.test.tsx#L9-L13) to begin with. When trying to render `@react-pdf/renderer` PDF content in Jest tests, I encounter errors like:

```
Error: BlobProvider is a web specific API. You're either using this component on Node, or your bundler is not loading react-pdf from the appropriate web build.
```

[Others do too and recommend mocking](https://github.com/diegomura/react-pdf/issues/710) as I've done

I created a follow-up issue for generating visual snapshots for PDFs generated by `@react-pdf/renderer`: https://github.com/votingworks/arlo/issues/1476

Reply:
Ah that's too bad